tensorflow/workspace2.bzl

```diff
@@ -177,6 +177,28 @@ def _tf_repositories():
         ],
     )
 
+    tf_http_archive(
+       name = "mkl_dnn_acl_compatible",
+       build_file = "//third_party/mkl_dnn:mkldnn_acl.BUILD",
+       sha256 = "5f7fd92e2d0bf83580656695d4404e2cd1390ecad36496fd8ba10b5adc905f70",
+       strip_prefix = "oneDNN-2.1",
```

Comment:
@penpornk, many thanks for the comments. I can also rebase this PR on the current TF master, what do you think about it?

Reply:
@alenik01 Thank you for the quick reply!
If we upgrade `mkl_dnn_v1` to newer than v2.1, would that still be fine with your code?

Reply:
Actually let's keep this separate repo (`mkl_dnn_acl_compatible`) for now, to be safe. Since the branch cut deadline is in 3 days. We should aim to get this PR in by today or tomorrow. (For any PR, there is a chance it breaks nightly tests and gets reverted the next day.)

Reply:
@penpornk,
>If we upgrade mkl_dnn_v1 to newer than v2.1, would that still be fine with your code?

yes, it will be fine. v2.2 is also a bit faster on AArch64.